.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,34 @@
-## Description
+# Description
 
 Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
 
 Fixes # (issue)
 
-## Tested on 
+## Type of change
+
+Please delete options that are not relevant.
+
```

Comment:
Since this is training labs, suggest adding one tab below

- [ ] New Exercise.